src/pybind/mgr/dashboard/services/ceph_service.py

```diff
@@ -120,12 +120,24 @@ def get_pool_list_with_stats(cls, application=None):
 
     @classmethod
     def get_pool_name_from_id(cls, pool_id):
+        pool = cls.get_pool_by_attribute('pool', pool_id)
+        return pool['pool_name'] if pool is not None else None
+
+    @classmethod
+    def get_pool_by_attribute(cls, attribute, value):
         pool_list = cls.get_pool_list()
         for pool in pool_list:
-            if pool['pool'] == pool_id:
-                return pool['pool_name']
+            if pool[attribute] == value:
+                return pool
         return None
 
+    @classmethod
+    def get_pool_pg_status(cls, pool_name):
```

Comment:
Please add type infos `# type (str) => str` or `# type (str) => list`. I'm currently confused what this method is returning.

Reply:
Type _hints_ are used with "single" arrow:

```python
def foo(text):
# type: (str) -> str
    return "asdf"
```

Reply:
Done :)